src/sentry/static/sentry/app/views/stream/searchBar.jsx

```diff
@@ -71,13 +75,33 @@ class SearchBar extends React.Component {
     );
   };
 
+  handleSearch = query => {
```

Comment:
We could push this logic into `<SmartSearchBar>` when we need to support Events search too...

Reply:
I'll just go ahead and do that now.